src/app/components/error/index.html

```diff
@@ -1,13 +1,196 @@
+          </sky-docs-visual-example-image>
+                
+        </sky-docs-visual-example>
+        
+        <h3 class="sky-text-danger">Don't use when</h3>
+        
+        <sky-docs-visual-example>
+            
+            <sky-docs-visual-example-text>
+              <p>Don’t use an error when there is an error with a background process. Information about background processes should use <a routerLink="/components/toast">toast</a> instead.</p>
+            </sky-docs-visual-example-text>
+            <sky-docs-visual-example-image>
+                <div class="sky-margin-stacked-content-section">
+                  <img src="~/assets/img/guidelines/error/error-usage-2.png" />
+                    <div class="caption caption-danger sky-text-danger">Don't use error for failed background processes.</div>
+                </div>
+                <div>
+                  <img src="~/assets/img/guidelines/error/error-usage-7.png" />
+                    <div class="caption caption-success sky-text-success">Do use a toast to inform the user about a process operating in the background.</div>
+                </div>
+            </sky-docs-visual-example-image>
+
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+            <p>Don’t use an error when the status of specific content needs user attention, but the system has not necessarily broken. To notify the user of a broken status on a workflow or record, use an <a routerLink="/components/alert">alert</a> instead.</p>
+        </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <div class="sky-margin-stacked-content-section">
+              <img src="~/assets/img/guidelines/error/error-usage-3.png" />
+                <div class="caption caption-danger sky-text-danger">Don't use the error component to represent issuse with content that user would expect to address.</div>
+            </div>
+              <div>
+              <img src="~/assets/img/guidelines/error/error-usage-4.png" />
+                <div class="caption caption-success sky-text-success">Do use alert to call attention to issues that are intended to prompt user interaction.</div>
+            </div>
+          </sky-docs-visual-example-image>
+                
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+          <p>Don’t use the security error type when a user doesn’t have permission to view a portion of the page such as a tile on a record page. Hide the tile from view instead.</p>
+        </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <div>
+              <img src="~/assets/img/guidelines/error/error-usage-5.png" />
+                <div class="caption caption-danger sky-text-danger">Don't use a security error in a portion of a page or modal. Hide the restricted content instead.</div>
```

Comment:
I'd change "the security error type" to "security error messages" and change "a user doesn't have" to "users lack." And I'd change "a portion" to "portions."